Added additional mkdocs dependencies, added gaming section

polats opened the pull request:
Hi @andytudhope , just wanted to say thanks again for the KERNEL syllabus 🙏  I've already learned a lot these past few days and am really looking forward to the incoming weeks!

I successfully set up a local repo of the docs, I had to install several markdown plugins to do so though. I added those plugins into the contribution doc.

I also added an initial gaming guild section and put up the image of our brainstorming session. I'm not sure where you want that displayed on the menus, so I just put it beside the resources for now. I was looking at putting it inside resources but didn't want to combo-break all the sections starting with Cs 😄 

Cheers and just let me know if anything needs to be changed!

polats committed:
add additional mkdocs plugin dependencies

polats committed:
added gaming guild section

andytudhope commented:
Gracias mi amor! :heart: My apologies on those dependencies - knew I was missing some stuff and was hoping the first victim would help me out... Thank you for taking the hit for all of us. You are the hero we don't deserve... What do you think about moving Gaming beneath resources and naming it `Curveball`, `Checkpoint`, `Cavort`?

I was, ofc, inspired by the 5 D's of Dodgeball: dodge, duck, dip dive, and dodge - so we can also repeat ourselves.... 

Also, it;s really not critical to keep the c's - just fun :wink:

polats committed:
Merge remote-tracking branch 'upstream/master'

polats committed:
moved gaming section under resources

polats commented:
No worries, it was an easy fix and am glad to have helped ❤️

I like keeping the C's! I've moved the gaming section into resources under Cavort (what a fun word), and added subsections:

- Checkpoint: Sort of the starting line or the homepage, showing the updates of the gaming guild per week, and,
- Curveball: a surprisingly delightful collection of links, including the projects of the gaming guild members and other gaming related links shared throughout the program
